### PR TITLE
chore(plot): add legend label to tooltips

### PR DESCRIPTION
**Pull request by hsgwa**
Signed-off-by: hsgwa <19860128+hsgwa@users.noreply.github.com>

## Description
this PR to add legend label to tooltips as the following picture shows.

Screenshot from 2022-12-20 10-30-44

## Related links

This PR is altenative of #218 

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/plot/visualize_lib/bokeh.py [hunk 1]
```diff
@@ -340,8 +345,13 @@ def _get_data_dict(
         target_object: TimeSeriesTypes
     ) -> Dict[str, Any]:
         data_dict: Dict[str, Any] = {}
+
         for k in self._get_source_keys(target_object):
             try:
+                if k == 'legend_label':
+                    label = self._legend.get_label(target_object)
+                    data_dict['legend_label'] = [f'legend_label = {label}']
+                    continue
```
Comment (atsushi421) [inline src/caret_analyze/plot/visualize_lib/bokeh.py:354]:
```suggestion
```

Comment (atsushi421) [reply src/caret_analyze/plot/visualize_lib/bokeh.py:354]: @hsgwa 
How about moving this process into the `_get_description` as follows?

```
def _get_description(
    self,
    key: str,
    target_object: TimeSeriesTypes
) -> str:
    if key == 'callback_param':
        if isinstance(target_object, TimerCallback):
            description = f'period_ns = {target_object.period_ns}'
        elif isinstance(target_object, SubscriptionCallback):
            description = f'subscribe_topic_name = {target_object.subscribe_topic_name}'
    elif key == 'legend_label':
        label = self._legend.get_label(target_object)
        description = f'legend_label = {label}'
    else:
        raise NotImplementedError()

    return description
```

**Review by atsushi421 — CHANGES_REQUESTED** [review]
@hsgwa 
Thank you for implementing this.
I have made one comment and would appreciate your confirmation.

File: src/caret_analyze/plot/visualize_lib/bokeh.py (hunk 1 above)
Comment (hsgwa) [reply src/caret_analyze/plot/visualize_lib/bokeh.py]: addressed in :[3bde03c](https://github.com/tier4/CARET_analyze/pull/222/commits/3bde03c234442de76964d4d7187775940ed8277b)

**Review by atsushi421 — APPROVED** [review]
LGTM!

**Pull request merged**